m4_sessions/1808-inning/prejudices.md

```diff
@@ -13,9 +16,11 @@ Turing's mission is to unlock human potential by training a diverse, inclusive s
 (What will students learn/leave with?): 
 
 ## Session Outline 
-
-(Please include who is facilitating and how much time you are going to spend in each section):
-
+* Intros (10-15 mins) led by entire group. Includes rules for how conversations will elvolve to make a safe space for discussion
```

Comment:
Yes!  Can you add the norms you want to use into this outline?  I would highly recommend you write the norms on the board or on chart paper so they are up the full session.